<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <resources><include location="../../../res/extui/extui.qrc"/></resources><author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>QG_DlgInsert</class>
  <widget class="QDialog" name="QG_DlgInsert">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>457</width>
        <height>310</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>1</hsizetype>
        <vsizetype>1</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>300</width>
        <height>190</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Insert</string>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <layout class="QHBoxLayout">
          <item>
            <layout class="QVBoxLayout">
              <item>
                <layout class="QHBoxLayout">
                  <item>
                    <widget class="QLabel" name="lLayer">
                      <property name="sizePolicy">
                        <sizepolicy>
                          <hsizetype>5</hsizetype>
                          <vsizetype>5</vsizetype>
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                        </sizepolicy>
                      </property>
                      <property name="text">
                        <string>Layer:</string>
                      </property>
                      <property name="wordWrap">
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QG_LayerBox" name="cbLayer"/>
                  </item>
                </layout>
              </item>
              <item>
                <widget class="QG_WidgetPen" name="wPen">
                  <property name="sizePolicy">
                    <sizepolicy>
                      <hsizetype>3</hsizetype>
                      <vsizetype>3</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <widget class="QGroupBox" name="buttonGroup8">
              <property name="title">
                <string>Geometry</string>
              </property>
              <layout class="QGridLayout">
                <item row="8" column="1">
                  <spacer name="spacer61">
                    <property name="sizeHint">
                      <size>
                        <width>20</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Vertical</enum>
                    </property>
                  </spacer>
                </item>
                <item row="8" column="0">
                  <spacer name="spacer58">
                    <property name="sizeHint">
                      <size>
                        <width>20</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Vertical</enum>
                    </property>
                  </spacer>
                </item>
                <item row="1" column="1">
                  <widget class="QLineEdit" name="leInsertionPointY">
                    <property name="minimumSize">
                      <size>
                        <width>64</width>
                        <height>0</height>
                      </size>
                    </property>
                  </widget>
                </item>
                <item row="0" column="1">
                  <widget class="QLineEdit" name="leInsertionPointX">
                    <property name="minimumSize">
                      <size>
                        <width>64</width>
                        <height>0</height>
                      </size>
                    </property>
                  </widget>
                </item>
                <item row="0" column="0">
                  <widget class="QLabel" name="lInsertionPointX">
                    <property name="text">
                      <string>Insertion point (x):</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="1" column="0">
                  <widget class="QLabel" name="lInsertionPointY">
                    <property name="text">
                      <string>Insertion point (y):</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="2" column="1">
                  <widget class="QLineEdit" name="leScale">
                    <property name="minimumSize">
                      <size>
                        <width>64</width>
                        <height>0</height>
                      </size>
                    </property>
                  </widget>
                </item>
                <item row="2" column="0">
                  <widget class="QLabel" name="lFactor">
                    <property name="text">
                      <string>Scale:</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="3" column="0">
                  <widget class="QLabel" name="lAngle">
                    <property name="text">
                      <string>Angle:</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="3" column="1">
                  <widget class="QLineEdit" name="leAngle">
                    <property name="minimumSize">
                      <size>
                        <width>64</width>
                        <height>0</height>
                      </size>
                    </property>
                  </widget>
                </item>
                <item row="4" column="0">
                  <widget class="QLabel" name="lRows">
                    <property name="text">
                      <string>Rows:</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="4" column="1">
                  <widget class="QLineEdit" name="leRows">
                    <property name="minimumSize">
                      <size>
                        <width>64</width>
                        <height>0</height>
                      </size>
                    </property>
                  </widget>
                </item>
                <item row="5" column="1">
                  <widget class="QLineEdit" name="leCols">
                    <property name="minimumSize">
                      <size>
                        <width>64</width>
                        <height>0</height>
                      </size>
                    </property>
                  </widget>
                </item>
                <item row="5" column="0">
                  <widget class="QLabel" name="lCols">
                    <property name="text">
                      <string>Columns:</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="6" column="0">
                  <widget class="QLabel" name="lRowSpacing">
                    <property name="text">
                      <string>Row Spacing:</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="6" column="1">
                  <widget class="QLineEdit" name="leRowSpacing">
                    <property name="minimumSize">
                      <size>
                        <width>64</width>
                        <height>0</height>
                      </size>
                    </property>
                  </widget>
                </item>
                <item row="7" column="1">
                  <widget class="QLineEdit" name="leColSpacing">
                    <property name="minimumSize">
                      <size>
                        <width>64</width>
                        <height>0</height>
                      </size>
                    </property>
                  </widget>
                </item>
                <item row="7" column="0">
                  <widget class="QLabel" name="lColSpacing">
                    <property name="text">
                      <string>Column Spacing:</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
              </layout>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <spacer name="spacer">
              <property name="sizeHint">
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="bOk">
              <property name="text">
                <string>&amp;OK</string>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="bCancel">
              <property name="text">
                <string>Cancel</string>
              </property>
              <property name="shortcut">
                <string>Esc</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <customwidgets>
    <customwidget>
      <class>QG_WidgetPen</class>
      <extends>QWidget</extends>
      <header location="local">qg_widgetpen.h</header>
      <sizehint>
        <width>-1</width>
        <height>-1</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
    <customwidget>
      <class>QG_LayerBox</class>
      <extends>QWidget</extends>
      <header location="local">qg_layerbox.h</header>
      <sizehint>
        <width>140</width>
        <height>22</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>7</hordata>
        <verdata>0</verdata>
      </sizepolicy>
      <pixmap>image1</pixmap>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>leInsertionPointX</tabstop>
    <tabstop>leInsertionPointY</tabstop>
    <tabstop>leScale</tabstop>
    <tabstop>leAngle</tabstop>
    <tabstop>leRows</tabstop>
    <tabstop>leCols</tabstop>
    <tabstop>leRowSpacing</tabstop>
    <tabstop>leColSpacing</tabstop>
    <tabstop>bOk</tabstop>
    <tabstop>bCancel</tabstop>
  </tabstops>
  <connections>
    <connection>
      <sender>bOk</sender>
      <signal>clicked()</signal>
      <receiver>QG_DlgInsert</receiver>
      <slot>accept()</slot>
    </connection>
    <connection>
      <sender>bCancel</sender>
      <signal>clicked()</signal>
      <receiver>QG_DlgInsert</receiver>
      <slot>reject()</slot>
    </connection>
  </connections>
</ui>
